### PR TITLE
Feat/app auth stacks

### PR DESCRIPTION
**Pull request by georgekwan**
## Why was this PR opened?
App/Auth stack has been setup

### CODE REVIEW

**Comment by huynhtk80** [comment]
User State needs to be:   const [user, setUser] = useState(true);
App stack shouldn't have additional <NavigationContainter>
App stack initialRouteName="either Questionform or test screen"

File: apps/Learning-Buddy/navigation/AppStack.jsx [hunk 1]
```diff
@@ -19,8 +19,6 @@ export const AppStack = () => {
           component={QuestionForm}
         ></Stack.Screen>
         <Stack.Screen name='Test Screen' component={TestScreen}></Stack.Screen>
-        <Stack.Screen name='Sign in' component={SignInScreen} />
-        <Stack.Screen name='Sign up' component={SignUpScreen} />
       </Stack.Navigator>
     </NavigationContainer>
```
Comment (huynhtk80) [inline apps/Learning-Buddy/navigation/AppStack.jsx:23]: Additional navigationcontainer causing render error, either remove or added independent property

File: apps/Learning-Buddy/navigation/RootNavigator.jsx [hunk 2]
```diff
@@ -0,0 +1,14 @@
+import React, { useState } from 'react';
+import { NavigationContainer } from '@react-navigation/native';
+import { AuthStack } from './AuthStack';
+import { AppStack } from './AppStack';
+
+export const RootNavigator = () => {
+  const { user } = useState();
```
Comment (huynhtk80) [inline apps/Learning-Buddy/navigation/RootNavigator.jsx:7]: Should be in a proper useState format eg.  const [user, setUser] = useState(initialstate);

**Review by huynhtk80 — CHANGES_REQUESTED** [review]

File: apps/Learning-Buddy/navigation/RootNavigator.jsx (hunk 2 above)
Comment (georgekwan) [reply apps/Learning-Buddy/navigation/RootNavigator.jsx]: fixed

File: apps/Learning-Buddy/navigation/AppStack.jsx (hunk 1 above)
Comment (georgekwan) [reply apps/Learning-Buddy/navigation/AppStack.jsx]: fixed

**Review by maggiechew — APPROVED** [review]

**Pull request merged**